src/js/main/index.js

```diff
@@ -330,14 +356,38 @@ $(function(){
                 response([]);
             });
         },
+        minLength:0, // required for empty input
         select: function(event, ui){
             $job_title.trigger('autocomplete-select', ui.item.label);
         },
-    });
+
+        // prevent disabled options from keyboard choosing
+        focus: function(event, ui){
+            var curr = $(event.currentTarget).find('.ui-state-active');
+            if(curr.parent().hasClass('ui-state-disabled')){
+                event.preventDefault();
+                curr.parent().siblings().first().children('div').mouseenter();  
+            }
```

Comment:
Can you change the variable `curr` into `$active_item` or some name? It is more readable